src/flows/recommendation_api/curated_corpus_candidates_flow.py

```diff
@@ -0,0 +1,76 @@
+from typing import Dict, Sequence
+import json
+import datetime
+
+from prefect import task, Flow, Parameter, unmapped
+from prefect.tasks.aws.s3 import S3List
+import pandas as pd
+import boto3
+from sagemaker.feature_store.feature_group import FeatureGroup, FeatureValue
+from sagemaker.session import Session
+
+from api_clients.pocket_snowflake_query import PocketSnowflakeQuery, OutputType
+from utils import config
+from utils.flow import get_flow_name, get_interval_schedule
+
+FLOW_NAME = get_flow_name(__file__)
+
+NEW_TAB_EN_US_CORPUS_CANDIDATE_SET_ID = 'deea0f06-9dc9-44a5-b864-fea4a4d0beb7'
+
+# Export approved corpus items by language and recency
+EXPORT_CORPUS_ITEMS_SQL = """
+SELECT
+    APPROVED_CORPUS_ITEM_EXTERNAL_ID as ID,
+    TOPIC
```

Comment:
`CORPUS_REVIEW_STATUS` can be `corpus` or `recommended` in this model, this might be useful information to preserve